plat/rockchip/rk3368/include/platform_def.h

```diff
@@ -74,6 +74,8 @@
 					 PLATFORM_CLUSTER_COUNT +	\
 					 PLATFORM_CORE_COUNT)
 
+#define PLATFORM_CLST_TO_CPUID_SHIFT	6
```

Comment:
But the PLATFORM_ prefix implies this is used by generic code. If you want to define a macro that is used by all Rockchip platforms, then a better prefix would be (for example) PLAT_RK_.